package.json

```diff
@@ -6,11 +6,33 @@
   "author": "Victoria Bernard <bernard.victoria22@gmail.com>",
   "license": "MIT",
   "dependencies": {
+    "@cloudflare/workers-types": "^2.0.0",
     "toml": "^3.0.0",
     "typescript": "^3.8.3"
   },
   "scripts": {
     "build": "cd workers-site && tsc -d",
-    "start": "npm run build && wrangler preview --watch"
+    "lint": "eslint './templates/**/*.ts' --cache --fix && prettier --write ./templates/**/*.{js,ts}",
+    "start": "npm run build && wrangler preview --watch",
+    "transpile": "tsc -p ./templates/typescript/tsconfig.json"
+  },
+  "devDependencies": {
+    "@typescript-eslint/eslint-plugin": "^2.33.0",
+    "@typescript-eslint/parser": "^2.33.0",
```

Comment:
these got a major version bump a few days after you put in this PR. might be worth updating to the latest.

Reply:
Done – only noteworthy change was setting `no-var: error` and `prefer-const: error` in the recommended ruleset.

https://github.com/typescript-eslint/typescript-eslint/pull/2001/files#diff-0259b6f21b7b64c48133978c53c66980R27-R28